server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/mailet/filter/JMAPFilteringTest.java

```diff
@@ -725,19 +726,19 @@ void mailDirectiveShouldSetLastMatchedRuleWhenMultipleRules(JMAPFilteringTestSys
                 Rule.builder()
                     .id(Rule.Id.of("1"))
                     .name("rule 1")
-                    .condition(Rule.Condition.of(SUBJECT, CONTAINS, UNSCRAMBLED_SUBJECT))
+                    .conditions(Arrays.asList(Rule.Condition.of(SUBJECT, CONTAINS, UNSCRAMBLED_SUBJECT)))
```

Comment:
We can keep a convenience method to build a rule with a single condition and avoid boiler plate.